main.tf

```diff
@@ -412,3 +412,18 @@ resource "aws_cloudwatch_log_group" "this" {
 
   tags = var.tags
 }
+
+################################################################################
+# Cluster Activity Stream
+################################################################################
+
+resource "aws_rds_cluster_activity_stream" "this" {
+  count = local.create && var.create_db_cluster_activity_stream ? 1 : 0
```

Comment:
There is a limitation on the activity stream as it's not supported by all the instances types.
works only on few of them per this doc : https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/DBActivityStreams.Overview.html 

how is that taken into account ?

Reply:
It's not - that's up to users